### ISSUE TITLE
Animated Statues can move when in view

### ISSUE THREAD

**Issue by ro5490** [BeeStation/BeeStation-Hornet#2900]
## Reproduction: Become Statue, move in client view

Tl Dr; 
`	for(var/atom/check in check_list)
		for(var/mob/living/M in viewers(world.view + 1, check) - src)
			if(M.client && CanAttack(M) && !M.has_unlimited_silicon_privilege)
				if(!M.eye_blind)
					return M
		for(var/obj/mecha/M in view(world.view + 1, check)) //assuming if you can see them they can see you
			if(M.occupant?.client)
				if(!M.occupant.eye_blind)
					return M.occupant`

`		for(var/mob/living/M in viewers(world.view + 1, check) - src)` Is the issue
it seems the 17x15 view changed fucked the `world.view + 1` value
Crossed suggested changing it to `+ "1x1"` instead however that seems to only work for the X axis and not the Y axis.

/code/modules/mob/living/simple_animal/hostile/statue.dm
line 108 to 134 is the codeblock

**Comment by ro5490** [comment]
Issue created by the change here - https://github.com/BeeStation/BeeStation-Hornet/pull/2668

**Comment by ike709** [comment]
@PowerfulBacon

**Comment by ro5490** [comment]
https://github.com/BeeStation/BeeStation-Hornet/pull/2911

This fix works, I'm closing the issue for now <3
